presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -88,12 +101,23 @@ Property Name                                        Description
 
 ==================================================== ============================================================
 
+Hadoop catalog
+^^^^^^^^^^^^^^
+
+Iceberg connector supports Hadoop as Catalog
+
 .. code-block:: none
 
     connector.name=iceberg
-    iceberg.catalog.type=nessie
-    iceberg.catalog.warehouse=/tmp
-    iceberg.nessie.uri=https://localhost:19120/api/v1
+    hive.metastore.uri=hostname:port
+    iceberg.catalog.type=hadoop
+
+.. note::
+
+    To use Hadoop as Catalog with Iceberg table, catalog requires ``hive.metastore.uri`` property
+    added even though it doesn't rely on metastore for metadata when using Hadoop Catalog. You
+    can provide a placeholder with ``hive.metastore.uri=thrift://localhost:9083``. This will not
+    be needed once check for ``hive.metastore.uri`` property is removed in case of Hadoop Catalog.
```

Comment:
```suggestion
    To use Hadoop as Catalog with Iceberg table, catalog requires the ``hive.metastore.uri`` property
    added even though it doesn't rely on metastore for metadata when using Hadoop Catalog. You
    can provide a placeholder with ``hive.metastore.uri=thrift://localhost:9083``. This will be fixed in a future release of Presto.
```

Is there an open issue for this BTW?

Reply:
Yes there are 2 issues reporting the same - https://github.com/prestodb/presto/issues/17342